src/components/layout/page-layout.tsx

```diff
@@ -13,21 +14,13 @@ const PageLayout: React.FC<PageLayoutProps> = ({ children, serverData }) => {
 
   return (
     <Block minHeight="100vh" display="flex" flexDirection={'column'}>
-      <Block
-        top={0}
-        left={0}
-        right={0}
-        position={'sticky'}
-        overrides={{
-          Block: {
-            style: () => ({
-              zIndex: 2,
-            }),
-          },
-        }}
-      >
-        <NavBar data={homepage.data.attributes.Header} />
-      </Block>
+      <LayersManager zIndex={2}>
```

Comment:
This semi defeats the purpose. The goal is we don't want to manage z-indexes and let layers manager take care of it - here and other places.

We need to think about rendering react in terms of layer. You have main layer that render pages and then another layer that render modals or other things

Reply:
Sir, I tried but unable to achieve layout what we need by this. Drawer by default takes higher z-index and we need to show navbar above drawer so we have to reduce z-index for that.